core/invoker/src/main/scala/whisk/core/container/WhiskContainer.scala

```diff
@@ -120,7 +120,7 @@ class WhiskContainer(
      * Tear down the container and retrieve the logs.
      */
     def teardown()(implicit transid: TransactionId): String = {
-        connection.foreach(_.close)
+        connection.foreach(_.close())
```

Comment:
@markusthoemmes i think this was missing from the new pool, added it for good measure.